arm_templates/logs.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
This should be YAML now, in fact the `.gitignore` file is set up to ignore all JSON in t his directory.

Once converted to yaml you can add some comments like where it's from, etc :smile: